airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/data/AirbyteValueIdentityMapper.kt

```diff
@@ -8,20 +8,45 @@ import io.airbyte.cdk.load.message.DestinationRecord
 import io.airbyte.protocol.models.v0.AirbyteRecordMessageMetaChange.Change
 import io.airbyte.protocol.models.v0.AirbyteRecordMessageMetaChange.Reason
 
+interface AirbyteValueMapper {
+    val meta: DestinationRecord.Meta
+    fun map(
+        value: AirbyteValue,
+        schema: AirbyteType,
+        path: List<String> = emptyList(),
+    ): AirbyteValue
```

Comment:
rather than directly mutating meta - would it make sense to return a value + list<change>, and have the caller merge the changes into an existing meta object? (or at least return `Pair<Value, Meta>`)

Reply:
That's more correct and could avoid the reflection issue. I can try it out as part of cleanup